src/main/resources/swagger.api/tenants.yaml

```diff
@@ -143,6 +144,13 @@ components:
         $ref: "schemas/common.yaml#/uuid"
       required: true
       description: The ID of consortium
+    adminUserId:
```

Comment:
@SerhiiNosko just curious, should it be query param or part of req body?

Reply:
I discussed with Serhi , it should be query param only.

Reply:
It should be request param.
It should not be part of req body because property adminUserId is not characterizes the tenant object and will violate OOP concepts if we add in req body